### PR TITLE
MT3-238 #done #comment Create a Header component

### PR DESCRIPTION
**Pull request by erbridge**
This is based on the existing implementation in `lbh-frontend`. It's missing a PNG image fallback, as it's not in spec, and we don't know where the user will be serving them from. Asking them to download an image separately, or have a build system for importing images feels like a bad solution.

The tests write to a snapshot file instead of inline, because the markup is very long.

The missing feature is tracked in #273.

See [`lbh-frontend`](https://github.com/LBHackney-IT/LBH-frontend/tree/master/src/lbh/components/lbh-header) for the reference component.

### CODE REVIEW

**Comment by CJWaszczuk** [comment]
Are we switching to non-inline snapshots for this because of the size of them?

File: src/components/icons/HackneyLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  /**
+   * Set to override the default height.
+   */
+  height?: string | null;
+}
+
+/**
+ * An icon to display in the start button on the start page of a service.
+ */
```
Comment (CJWaszczuk) [inline src/components/icons/HackneyLogo.tsx]: Guessing this should be for the logo

**Comment by erbridge** [comment]
> Are we switching to non-inline snapshots for this because of the size of them?

Yup. See the PR description :).

File: src/components/icons/HackneyLogo.tsx (hunk 1 above)
Comment (erbridge) [reply src/components/icons/HackneyLogo.tsx]: Oops. Yes. Copy and paste error...

File: src/components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+   * The navigation links to include in the header.
+   */
+  children?: React.ReactNode | null;
+}
+
+/**
+ * A component that wraps a set of elements within a single container. Can be
+ * used to provide consistency in layout, for example, for all elements on a
+ * page to have a certain width.
+ */
```
Comment (CJWaszczuk) [inline src/components/Header.tsx]: This also

**Review by CJWaszczuk — APPROVED** [review]

**Pull request merged**